src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildWithATargetPlatform.cs

```diff
@@ -63,6 +63,28 @@ public void It_defines_target_platform_from_target_framework(string targetFramew
             assertValue("TargetPlatformDisplayName", $"{expectedTargetPlatformIdentifier} {expectedTargetPlatformVersion}");
         }
 
+        [WindowsOnlyRequiresMSBuildVersionFact("16.8.0.41402")]
+        public void It_defines_target_platform_from_target_framework_with_explicit_version()
+        {
+            var targetPlatformVersion = "";
```

Comment:
Shouldn't `targetPlatformVersion` be an actual value instead of an empty string?